### PR TITLE
Updated calls to backend to only get needed fields

### PR DESCRIPTION
**Pull request by DeclanLacey**
Changed how data was received from the backend to ensure that only the needed fields were grabbed. This makes the website more secure as we do not have additional data (such as emails and phone numbers) being brought down and visible in the network tab. This will likely also make the website more performant.

### CODE REVIEW

**Comment by DeclanLacey** [comment]
I also updated the automation script in Airtable, as I had to change the names of some of the fields to be in camel case in order for this solution to work. We will need to run more tests on the automation to ensure that everything still works correctly.

File: src/components/RescueDetails.tsx [hunk 1]
```diff
@@ -278,10 +291,9 @@ export default function RescueDetails({id}: { id: string }) {
 
     // this is what actually populates the list
     function populateNameOptions() {
-        const volunteerOptions = volunteers.filter((vol: {
-            id: string,
-            name: string
-        }) => birdRescue!.possibleVolunteers.includes(vol.id) && vol.name !== birdRescue?.currentVolunteer)
+        
+        const volunteerOptions = volunteers.filter((vol: {id: string, name: string}) => birdRescue!.possibleVolunteers.includes(vol.id) && vol.name !== birdRescue?.currentVolunteer)
+        console.log(volunteerOptions)
```
Comment (travissanderson) [inline src/components/RescueDetails.tsx:296]: can remove this :)

File: src/components/BirdAlertList.tsx [hunk 2]
```diff
@@ -46,30 +46,54 @@ export default function BirdAlertList() {
...
+                        secondVolunteer: fields.SecondVolunteer as string,
+                        twoPersonRescue: fields.TwoPersonRescue as Boolean,
+                        photo: fields.BirdPhoto ? ((fields.BirdPhoto as object[])[0] as {
+                            url: string,
+                            width: number,
+                            height: number
+                        }) : {} as { url: string, width: number, height: number },
+                    })
+                })
+                processNextPage()
```
Comment (travissanderson) [inline src/components/BirdAlertList.tsx:95]: nice

**Comment by DeclanLacey** [comment]
Removed the console.log and I had forgotten to change the names of the fields when getting the bird alert details, should be fixed now.

File: src/components/BirdAlertList.tsx (hunk 2 above)
Comment (DeclanLacey) [reply src/components/BirdAlertList.tsx:95]: Thanks! Airtable did not make it easy to find a way to do this. I was finally able to find a random forum post where someone showed some similar code.

**Comment by travissanderson** [comment]
looks good to me, but I am not planning to test it so that is up to you :)

**Comment by DeclanLacey** [comment]
Yeah I will merge this for now, but me and Jen are having a meeting at some point today or tomorrow. So will probably test it then. Thanks Travis! @travissanderson

**Pull request merged**